### PR TITLE
number agreement in authenticators-users-basics

### PR DESCRIPTION
**Pull request by TaofeeqatDev**
1. Fix some typo errors
2. Fix some grammar errors

### CODE REVIEW

File: docs/source/getting-started/authenticators-users-basics.md [hunk 1]
```diff
@@ -3,12 +3,12 @@
...
 
 The default Authenticator uses [PAM][] (Pluggable Authentication Module) to authenticate system users with
-their username and password. With the default Authenticator, any user
+their usernames and passwords. With the default Authenticator, any user
 with an account and password on the system will be allowed to login.
 
 ## Create a set of allowed users (`allowed_users`)
 
-You can restrict which users are allowed to login with a set,
+You can restrict which users are allowed to login with a set
```
Comment (minrk) [inline docs/source/getting-started/authenticators-users-basics.md:11]: Can we get this comma back? It was correct before.

File: docs/source/getting-started/authenticators-users-basics.md [hunk 2]
```diff
@@ -90,7 +90,7 @@ file is:
 c.LocalAuthenticator.create_system_users = True
 ```
 
-Adding a user to the Hub that doesn't already exist on the system will
+Adding a user to the Hub that doesn't exist already on the system will
```
Comment (minrk) [inline docs/source/getting-started/authenticators-users-basics.md:93]: I think `already exist` is more natural here.

File: docs/source/getting-started/authenticators-users-basics.md [hunk 3]
```diff
@@ -3,12 +3,12 @@
 # Authentication and User Basics
 
 The default Authenticator uses [PAM][] (Pluggable Authentication Module) to authenticate system users with
-their username and password. With the default Authenticator, any user
+their usernames and passwords. With the default Authenticator, any user
```
Comment (minrk) [inline docs/source/getting-started/authenticators-users-basics.md:6]: 👍

**Review by minrk — COMMENTED** [review]
Thanks! I think a couple of the changes were better before, if you could put those back we can merge.

File: docs/source/getting-started/authenticators-users-basics.md (hunk 1 above)
Comment (TaofeeqatDev) [reply docs/source/getting-started/authenticators-users-basics.md:11]: Alright, I will do that right away.

File: docs/source/getting-started/authenticators-users-basics.md (hunk 2 above)
Comment (TaofeeqatDev) [reply docs/source/getting-started/authenticators-users-basics.md:93]: Ok, I understand. I will make the necessary changes.

**Comment by minrk** [comment]
Thanks!

**Pull request merged**